api/lib/domain/models/PixEdu1erDegreBadgeAcquisitionOrderer.js

```diff
@@ -0,0 +1,40 @@
+const {
+  PIX_EDU_FORMATION_INITIALE_1ER_DEGRE_INITIE,
+  PIX_EDU_FORMATION_INITIALE_1ER_DEGRE_CONFIRME,
+  PIX_EDU_FORMATION_CONTINUE_1ER_DEGRE_CONFIRME,
+  PIX_EDU_FORMATION_CONTINUE_1ER_DEGRE_AVANCE,
+  PIX_EDU_FORMATION_CONTINUE_1ER_DEGRE_EXPERT,
+} = require('./Badge').keys;
+
+class PixEdu1erDegreBadgeAcquisitionOrderer {
+  constructor({ badgesAcquisitions } = {}) {
+    this.badgesAcquisitions = badgesAcquisitions;
+  }
+
+  getHighestBadge() {
+    const formateurFormationContinueBadgeAcquisition = this.badgesAcquisitions.find(
+      (badgesAcquisition) => badgesAcquisition.badgeKey === PIX_EDU_FORMATION_CONTINUE_1ER_DEGRE_EXPERT
```

Comment:
Par contre je comprends pas bien toute cette peine que tu as pris pour faire du renommage pour avoir au final un truc dissonnant 
const **formateurFormationContinueBadgeAcquisition** = this.badgesAcquisitions.find(
      (badgesAcquisition) => badgesAcquisition.badgeKey === **PIX_EDU_FORMATION_CONTINUE_1ER_DEGRE_EXPERT**